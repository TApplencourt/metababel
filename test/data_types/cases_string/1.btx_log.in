event: { pf_1 = "", pf_2 = "event: { pf_1 = \"\" }", pf_3 = "hyEhk9uaOQJb = 8HT+0u/lu6rGq4ivNLx8HSYaiWvVfRW6fkDQX16FgJtv7mIHRAQW0mLUnYySLEB0X/ptpitF+RuAfWv6hWy/pW/CFpdbiESVPxB7nm8zgeTWfORTPKTRZZ0yNrcHMDdDjTlaEHl+eh+0drHekW0GyAGyioKx7wEtdYYyHJP1sOjWQDxEeXcmvvBJRsRvpnbMYh GVtq5YJ   JLCZs/ENyt0e9bmN5jCbgKUauqRr78Jx8L6uonBFueYr2ge6ibjSWM=" }
