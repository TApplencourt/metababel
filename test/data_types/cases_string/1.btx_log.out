Trace class:
  Stream class `sc` (ID 0):
    Supports packets: No
    Supports discarded events: No
    Supports discarded packets: No
    Event class `event` (ID 0):
      Payload field class: Structure (3 members):
        pf_1: String
        pf_2: String
        pf_3: String

{Trace 0, Stream class ID 0, Stream ID 0}
Stream beginning:
  Class name: sc
  Trace:
    Stream (ID 0, Class ID 0)

Trace class:
  Stream class `sc` (ID 0):
    Supports packets: No
    Supports discarded events: No
    Supports discarded packets: No
    Event class `event` (ID 0):
      Payload field class: Structure (3 members):
        pf_1: String
        pf_2: String
        pf_3: String

{Trace 1, Stream class ID 0, Stream ID 0}
Stream beginning:
  Class name: sc
  Trace:
    Stream (ID 0, Class ID 0)

{Trace 0, Stream class ID 0, Stream ID 0}
Event `event` (Class ID 0):
  Payload:
    pf_1: 
    pf_2: event: { pf_1 = "" }
    pf_3: hyEhk9uaOQJb = 8HT+0u/lu6rGq4ivNLx8HSYaiWvVfRW6fkDQX16FgJtv7mIHRAQW0mLUnYySLEB0X/ptpitF+RuAfWv6hWy/pW/CFpdbiESVPxB7nm8zgeTWfORTPKTRZZ0yNrcHMDdDjTlaEHl+eh+0drHekW0GyAGyioKx7wEtdYYyHJP1sOjWQDxEeXcmvvBJRsRvpnbMYh GVtq5YJ   JLCZs/ENyt0e9bmN5jCbgKUauqRr78Jx8L6uonBFueYr2ge6ibjSWM=

{Trace 1, Stream class ID 0, Stream ID 0}
Stream end

{Trace 0, Stream class ID 0, Stream ID 0}
Stream end
